src/local-storage-assignment-cache.ts

```diff
@@ -0,0 +1,51 @@
+import { AssignmentCache } from '@eppo/js-client-sdk-common';
+
+import { hasWindowLocalStorage } from './local-storage';
+
+class LocalStorageAssignmentShim {
+  LOCAL_STORAGE_KEY = 'EPPO_LOCAL_STORAGE_ASSIGNMENT_CACHE';
+
+  public has(key: string): boolean {
+    if (!hasWindowLocalStorage()) {
+      return false;
```

Comment:
Thought about throwing an error here however there's nothing with people disabling this API and functionally the SDK should continue to work.

Reply:
Makes sense to me too. No error throwing here 👍